examples/milestone/TicTacToe/Logic/Game.juvix

```diff
@@ -69,6 +69,16 @@ inductive Square {
   occupied : Symbol → Square;
 };
 
+--- A row in a board implemented as ;List Square;
+---
+--- >>> Row;
+Row : Type;
```

Comment:
This should be commented out?

If I use the `Row` type alias in the program and compile it I get:

```
juvix: TypeUniverse Micro to C: not yet supported
CallStack (from HasCallStack):
  error, called at src/Juvix/Prelude/Base.hs:242:9 in juvix-0.2.2-Bjv4Lkpbnpb5ZXqkLcn1KX:Juvix.Prelude.Base
  error, called at src/Juvix/Compiler/Backend/C/Data/Base.hs:23:19 in juvix-0.2.2-Bjv4Lkpbnpb5ZXqkLcn1KX:Juvix.Compiler.Backend.C.Data.Base
```
(limitation of the current C backend).

Reply:
done